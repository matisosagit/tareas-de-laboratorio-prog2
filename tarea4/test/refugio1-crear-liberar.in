crearRefugio 100
liberarRefugio
Fin