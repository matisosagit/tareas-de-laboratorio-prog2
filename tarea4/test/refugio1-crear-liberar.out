1> Refugio creado.
2> Se liberÃ³ la memoria asociada al refugio.
3> Fin.
